### PR TITLE
feat: add text theme

### PR DESCRIPTION
**Pull request by gaygoblin**
# text
a TUI theme based on the one of the same name from Spicetify.
Most of the code was written by me. 
This is my first "*big*" CSS project, and I really liked making it :3
#
- [x] I understand and have followed the [contribution guide](https://github.com/revoltchat/.github/blob/master/.github/CONTRIBUTING.md)
- [x] I have tested my changes locally and they are working as intended

### CODE REVIEW

File: data/text/preset.toml
Comment (Rexogamer) [inline data/text/preset.toml:1]: rename this to `Preset.toml` (capitalised)

**Review by Rexogamer — CHANGES_REQUESTED** [review]

File: data/text/preset.toml
Comment (gaygoblin) [reply data/text/preset.toml:1]: Done.

File: data/text/Preset.toml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+slug = "text"
+name = "text"
+creator = "violet"
+commit = "e8a8229"
+desc = "a tui theme based on the "text" theme from spicetify"
```
Comment (Rexogamer) [inline data/text/Preset.toml:5]: escape the quotes:

```suggestion
desc = "a tui theme based on the \"text\" theme from spicetify"
```

File: data/text/Preset.toml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+success = "#65e572"
+warning = "#faa352"
+tooltip = "#2e2837"
+
+hover = "rgba(0, 0, 0, 0.1)"
+scrollbar-thumb = "rgba(0,0,0,0)"
+scrollbar-track = "rgba(0,0,0,0)"
+
+primary-background = "#2e2837"
+primary-header: "#1e1e1e"
```
Comment (Rexogamer) [inline data/text/Preset.toml:26]:
```suggestion
primary-header = "#1e1e1e"
```

**Comment by gaygoblin** [comment]
why isnt the changes requested thing done i resolved the change

File: data/text/Preset.toml [hunk 3]
```diff
@@ -0,0 +1,42 @@
+slug = "text"
+name = "text"
+creator = "violet"
+commit = "e8a8229"
+desc = "a tui theme based on the \"text\" theme from spicetify"
```
Comment (Rexogamer) [inline data/text/Preset.toml:5]:
```suggestion
description = "a tui theme based on the \"text\" theme from spicetify"
```

**Review by Rexogamer — CHANGES_REQUESTED** [review]

**Comment by gaygoblin** [comment]
okay! everything _should_ be fine now.. i double checked the preset.toml again and it seems good!

**Comment by Rexogamer** [comment]
still not quite, but it's getting closer. you might want to look at [this](https://github.com/revoltchat/themes/blob/master/data/amethyst/Preset.toml) for a working example - note the use of sections instead of certain key names (e.g. `[variables.primary]` with a `background` key instead of `primary-background`).

alternatively, you can use [this tool](https://rexovolt.github.io/revolt-theme-tools/) I made to convert the original JSON theme into a working TOML file. hope that helps :3

**Pull request closed without merging**